modules/db/src/main/java/org/opencastproject/db/Activator.java

```diff
@@ -137,6 +139,36 @@ else if (pooledDataSource.getMaxIdleTime() == 0) {
     logger.info("Database connection pool established at {}", jdbcUrl);
     logger.info("Database connection pool parameters: max.size={}, min.size={}, max.idle.time={}",
       pooledDataSource.getMaxPoolSize(), pooledDataSource.getMinPoolSize(), pooledDataSource.getMaxIdleTime());
+    Statement statement = pooledDataSource.getConnection().createStatement();
+
+    long time = System.currentTimeMillis();
```

Comment:
I think using a unique table name is a good solution for this problem. But imaging a cluster to start up at the same time. That would potentially select the same table names. Maybe just pick a large random number instead?